src/handlers/commandHandler.js

```diff
@@ -142,6 +143,10 @@ export default class CommandHandler {
       data = json.eu;
     } else if (this.region === 'kr') {
       data = json.kr;
+    } else if (this.region === 'psn') {
+      data = json.any;
+    } else if (this.region === 'xbl') {
```

Comment:
Could you convert this to `else if ((this.region == 'xbl') || (this.region == 'psn'))` as well?